src/Microsoft.Health.Fhir.Core/Features/Operations/DataConvert/Models/ContainerRegistryInfo.cs

```diff
@@ -0,0 +1,16 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.DataConvert.Models
+{
+    public class ContainerRegistryInfo
+    {
+        public string ContainerRegistryServer { get; set; } = string.Empty;
+
+        public string ContainerRegistryUsername { get; set; } = string.Empty;
+
+        public string ContainerRegistryPassword { get; set; } = string.Empty;
```

Comment:
Are there any other ways to access the container registry other than username/password?  Is Managed Identity an option?

Reply:
Yes. For PaaS solution, we can use managed identity as we are expecting users to use ACR. For the OSS solution, we don't want to limit customers to choose ACR, thus we cannot use managed identity with other customized container registries.

Reply:
Do we have any other secrets like this in OSS?

Reply:
We have some other keys in the [appsettings.json](https://github.com/microsoft/fhir-server/blob/master/src/Microsoft.Health.Fhir.Shared.Web/appsettings.json) like connection string in the exportConfig, cosmos db key, application insight key, etc.